toolkit/themes/shared/design-system/config.js

```diff
@@ -38,24 +50,30 @@ function hcmFormatter(args) {
  * @param {string} [tokenArgs.mediaQuery]
  *  Media query formatted CSS should be wrapped in. This is used
  *  to determine what property we are parsing from the token values.
+ * @param {string} [tokenArgs.surface]
+ *  Specifies a desktop surface, either "brand" or "platform".
  * @param {object} tokenArgs.args
  *  Formatter arguments provided by style-dictionary. See more at
  *  https://amzn.github.io/style-dictionary/#/formats?id=formatter
  * @returns {string} Tokens formatted into a CSS string.
  */
-function formatTokens({ mediaQuery, args }) {
-  let prop = MEDIA_QUERY_PROPERTY_MAP[mediaQuery] ?? "value";
+function formatTokens({ mediaQuery, surface, args }) {
+  let prop = MEDIA_QUERY_PROPERTY_MAP[mediaQuery] ?? "default";
   let dictionary = Object.assign({}, args.dictionary);
   let tokens = [];
 
   dictionary.allTokens.forEach(token => {
-    let value = token[prop] || token.original.value[prop]
-    if (value && typeof value !== "object") {
-      let formattedToken = transformTokenValue(token, prop, dictionary);
+    let originalVal = getOriginalTokenValue(token, prop, surface);
+    if (originalVal) {
+      let formattedToken = transformTokenValue(token, originalVal, dictionary);
       tokens.push(formattedToken);
     }
   });
 
+  if (!tokens.length) {
+    return "";
+  }
```

Comment:
Just so I understand, this handles these cases:
- An invalid surface was passed in to formatTokens()
- There's no tokens present for the particular surface

Is there anything else I'm missing here?

Reply:
Yeah so this could happen in a few cases related to either the surface or the media query prop. The case we're actually hitting right now is we don't have any media query tokens for the brand and platform surfaces.

Reply:
Ah okay, thanks for the clarification!